lisa/sut_orchestrator/azure/features.py

```diff
@@ -596,6 +600,103 @@ def get_raw_data_disks(self) -> List[str]:
             disk_array[int(disk.split("/")[-1].replace("lun", ""))] = cmd_result.stdout
         return disk_array
 
+    def add_managed_data_disk(
```

Comment:
We can provide an abstract method in LISA level feature like `def add_data_disk(count:int, disk_type:DiskType, size_in_gb:int=20) -> List[str]`. The disk type parameter is platform specific, but we can reuse the DiskType in schema.py so far.

The names aren't matter, so we can decide it in the method and return it later. The luns can be checked in this method.

Reply:
Are luns valid outside of azure or are they azure specific?

Reply:
I don't know, it maybe appliable to all platforms. But it doesn't need to be calculated at test case level.